app/routes/wow.shopping-list.tsx

```
@@ -63,44 +77,114 @@
+    return json({
+      ...(await result.json()),
+      sortby: 'discount',
+      formValues: validatedFormData.data
+    })
+  }
+
+  return json({})
+}
+
+type LoaderResponseType =
+  | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)